### ISSUE TITLE
Atom showing me unreadable error messages when running Hydrogen

### ISSUE THREAD

**Issue by febbbb** [nteract/hydrogen#2054]
### Prerequisites

* [√] Have you checked the solutions in our [troubleshooting guide](https://nteract.gitbooks.io/hydrogen/docs/Troubleshooting.html) for common problems ?
* [√ ] Have you checked that your issue [isn't already filed](https://github.com/nteract/hydrogen/issues) ?

### Description

When I open Atom and try to run any code for the first time, Hydrogen show me the Kernels are updated, but immediately give me error messages with unreadable words. 
After that, if I try to run any code again, nothing will happen. 

### Steps to Reproduce
 
As described:
Step 1: Open Atom
Step 2: Write any code
Step 3: Select code and run with Hydrogen

### Versions

OS version:
[Your OS's version]
Windows 7

> `atom --version`

```
atom 1.54.0
```

> `apm --version`

```
apm 2.5.2
```

Hydrogen version:

[Your Hydrogen's version]
hydrogen 2.14.7

### Plugins

Have you installed and activated any of the Hydrogen plugins below ?
- [√ ] hydrogen-python
- [√] Hydrogen Launcher
- [√ ] Data Explorer

### Expected behavior

Code to run

### Screenshots

![11](https://user-images.githubusercontent.com/78739657/107230677-532d2300-6a5a-11eb-8cc0-ad289371731d.jpg)
![1](https://user-images.githubusercontent.com/78739657/107230730-62ac6c00-6a5a-11eb-8390-0224f1df0bb6.PNG)

### Additional context

[Add any other context about the problem here.]

**Comment by febbbb** [comment]
> Given that the error happens after updating the list of kernels, the output of running `jupyter kernelspec list` could be helpful to find the cause.

Thanks for replying. I'm new to python and Atom. Do you mind to let me know the step by step method of how to run the jupyter kernelspec list? I had been using Hydrogen to run my codes in Atom and with it not working I am not sure how to run it

**Comment by n-riesco** [comment]
`jupyter kernelspec list` needs to be run inside a terminal. It looks like you're on Windows. To open the terminal, press WindowKey+R and type `cmd`.

**Comment by febbbb** [comment]
> `jupyter kernelspec list` needs to be run inside a terminal. It looks like you're on Windows. To open the terminal, press WindowKey+R and type `cmd`.

![1](https://user-images.githubusercontent.com/78739657/107239978-5cbb8880-6a64-11eb-8009-3e2e33ca598f.JPG)

Do I just key in python and then jupyter kernelspec list? It shows error like this.

**Comment by n-riesco** [comment]
No need to run 'python`. Instead, type `jupyter kernelspec list`.

**Comment by febbbb** [comment]
> No need to run 'python`. Instead, type `jupyter kernelspec list`.

If I tried directly type `jupyter kernelspec list`, it will show me error message:
`'jupyter' is not recognized as an internal or external command, operable command or batch file`

**Comment by n-riesco** [comment]
That's unexpected. How did you install the Python kernel?

**Comment by n-riesco** [comment]
Could you try `ipython kernelspec list` instead?

**Comment by febbbb** [comment]
> That's unexpected. How did you install the Python kernel?

Hi, thanks so much for helping. I have finally solved this jupyter error and now if I type `jupyter kernelspec  list`, it will show me this:
![12](https://user-images.githubusercontent.com/78739657/107260958-aa8ebb80-6a79-11eb-9f02-5d2dcbdcdfcb.JPG)

**Comment by n-riesco** [comment]
The path to the kernel specs look OK (I thought the cause of the bug could be there).

Have you checked again if Hydrogen is still failing?

**Comment by febbbb** [comment]
Hi, yes it is still showing me this unreadable message
![123](https://user-images.githubusercontent.com/78739657/107265198-f09a4e00-6a7e-11eb-8550-8c94b1f1dfe9.JPG)

**Comment by n-riesco** [comment]
I'll try to reproduce the issue on a Windows machine.
What method did you use to install Python and Jupyter in your machine?

**Comment by febbbb** [comment]
> I'll try to reproduce the issue on a Windows machine.
> What method did you use to install Python and Jupyter in your machine?

Thank you so much for helping. I have solved this problem by restoring my whole system to before I first downloaded python, to make sure my re-installation will start afresh, and re-installing everything again. 

Also for your question just for reference I installed python by downloading python 3.8.exe directly from their website. I installed Jupyter by using Anaconda.

Thanks again for helping!!
